openssl/src/asn1.rs

```diff
@@ -51,6 +51,29 @@ impl fmt::Display for Asn1TimeRef {
     }
 }
 
+impl Asn1TimeRef {
+    #[cfg(any(ossl102, ossl110))]
+    pub fn as_unix(&self) -> Result<time_t, ErrorStack> {
+        Asn1Time::from_unix(0)?.diff(Some(self))
+    }
+
+    #[cfg(any(ossl102, ossl110))]
+    pub fn diff(&self, compare: Option<&Self>) -> Result<time_t, ErrorStack> {
```

Comment:
Then `as_unix` would be difficult to implement, also a constant like this is used here: https://github.com/sfackler/rust-openssl/blob/master/openssl/src/asn1.rs#L66